frontend/pages/hosts/ManageHostsPage/HostsPageConfig.tsx

```diff
@@ -2,6 +2,12 @@ import React from "react";
 
 import Icon from "components/Icon";
 
+// the source of truth for the filter option names.
+// there are used on many other pages but we define them here.
+// TODO: add other filter options here. Also look to define this in
+// the host entities file later.
+export const DISK_ENCRYPTION_QUERY_PARAM_NAME = "os_settings_disk_encryption";
```

Comment:
I like the idea of defining the constants being used for other one-off param names, like `inherited_order_key` in the manage policies name. Might be a good discussion.